development/nautobot_config.py

```diff
@@ -1,11 +1,31 @@
-"""Nautobot configuration."""
+"""Nautobot config."""
+# pylint: disable=invalid-envvar-default
+#########################
+#                       #
+#   Required settings   #
+#                       #
+#########################
+
 import os
 import sys
 
 from distutils.util import strtobool
-
 from django.core.exceptions import ImproperlyConfigured
-from nautobot.core.settings import *  # noqa: F401,F403 pylint: disable=wildcard-import,unused-wildcard-import
```

Comment:
Need to keep this for pylint-django to be able to properly load all the settings and not crash.